tests/terraform/graph/graph_builder/test_oci_policy.py

```diff
@@ -1,43 +1,38 @@
+"""Test if OCI policy statements are evaluated correctly."""
+
 import os
-import unittest
+from pathlib import Path
+from unittest import mock
 
-from checkov.common.graph.db_connectors.networkx.networkx_db_connector import (
-    NetworkxConnector,
-)
+from checkov.common.graph.db_connectors.networkx.networkx_db_connector import NetworkxConnector
 from checkov.terraform.graph_manager import TerraformGraphManager
 
+@mock.patch.dict(os.environ, {"CKV_TERRAFORM_PROVIDER": "OCI"})
+def test_oci_policy_statements_with_provider_env_var():
```

Comment:
I followed the other tests and created a class to be consistent. But if a method is enough, I'm happy with that.

Reply:
yeah it is kind of mixed. we started with `untitest` style and still do them, but as a test runner we use `pytest` and then you actually can just use functions, which I prefer 🙂